incubator/busbar/templates/deployment.yaml

```diff
@@ -0,0 +1,171 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "busbar.fullname" . }}-web
+  labels:
+    app: {{ template "busbar.fullname" . }}-web
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "busbar.fullname" . }}-web
```

Comment:
After changing the `app` label as suggested above, you'll have to add the `release` label to labels selectors. Apply everywhere.